### ISSUE TITLE
Section of path that starts with a number results in Bolt error

### ISSUE THREAD

**Issue by mm-gmbd** [firebase/bolt#119]
A path with a particular section (i.e. that is delimited with a forward-slash) that begins with a number produces the "generic" error:

```
bolt:LINE_NUM:COL_NUM: Expected comment, end of input, function definition, path statement, type statement or whitespace but "p" found.
```

So the following path does not work:

```
path /a/path/to/1numberedObject {
  read() = true
}
```

I tried enclosing the entire path in quotes, but that didn't do the trick. Suggestions? I know Firebase keys can begin with numbers, but they must be stored as strings...

**Comment by mckoss** [comment]
I think this was fixed in Bolt 0.7.0 (or 0.7.1) - it works for me.  Can you get the latest from npm and confirm? @mm-gmbd

**Comment by mckoss** [comment]
I see the problem.  Bolt 0.7 is more strict about semicolons - you're missing one in the read() expression.  This is relaxed in 0.7.1 (this was unrelated to the path statement).
